doc/howto/network_bridge_firewalld.md

```diff
@@ -1,52 +1,120 @@
+Therefore, rules in one namespace can still affect rules in another namespace, and firewall applications can still impact LXD network functionality.
+
+If your system supports and uses nftables, LXD detects this and switches to nftables mode.
+In this mode, LXD adds its rules into the nftables, using its own nftables namespace.
+
+### Disable LXD's firewall rules
+
+Run the following commands to prevent LXD from setting firewall rules for a specific network bridge (for example, `lxdbr0`):
+
+    lxc network set <network_bridge> ipv4.nat false
+    lxc network set <network_bridge> ipv6.nat false
+    lxc network set <network_bridge> ipv6.firewall false
+    lxc network set <network_bridge> ipv4.firewall false
+
+### Firewalld: Add LXD firewall rules
+
+To configure the LXD firewall rules in firewalld, you must add input, output, forward and postrouting rules for the network bridge (for example, `lxdbr0`) and the associated IP range (for example, `192.0.2.0/24`):
+
+    sudo firewall-cmd --permanent --direct --add-rule ipv4 filter INPUT 0 -i <network_bridge> -s <IP_range> -m comment --comment "generated by firewalld for LXD" -j ACCEPT
+    sudo firewall-cmd --permanent --direct --add-rule ipv4 filter OUTPUT 0 -o <network_bridge> -d <IP_range> -m comment --comment "generated by firewalld for LXD" -j ACCEPT
+    sudo firewall-cmd --permanent --direct --add-rule ipv4 filter FORWARD 0 -i <network_bridge> -s <IP_range> -m comment --comment "generated by firewalld for LXD" -j ACCEPT
+    sudo firewall-cmd --permanent --direct --add-rule ipv4 nat POSTROUTING 0 -s <IP_range> ! -d <IP_range> -m comment --comment "generated by firewalld for LXD" -j MASQUERADE
+    sudo firewall-cmd --reload
+
+Use the following command to check that the rules are taken into account by firewalld:
+
+    sudo firewall-cmd --direct --get-all-rules
+
+<!-- Include start warning -->
+```{warning}
+The commands given above show a simple example configuration.
+Depending on your use case, you might need more advanced rules and the example configuration might inadvertently introduce a security risk.
 ```
-firewall-cmd --direct --get-all-rules
+<!-- Include end warning -->
+
+### UFW: Add LXD firewall rules
+
+To configure the LXD firewall rules in UFW, you must allow incoming, outgoing and routing traffic for the network bridge (for example, `lxdbr0`) and the associated IP range (for example, `192.0.2.0/24`):
+
+    sudo ufw allow in on <network_bridge> # allow DHCP/DNS/IPv6 RA
+    sudo ufw route allow in on <network_bridge> # allow routing from bridge network to the external network
+
+% Repeat warning from above
+```{include} network_bridge_firewalld.md
+    :start-after: <!-- Include start warning -->
+    :end-before: <!-- Include end warning -->
 ```
 
-Warning: what is exposed above is not a fool-proof approach and may end up inadvertently introducing a security risk.
+## LXD and Docker
```

Comment:
I'm happy to see this, but can @stgraber check this is OK to have in our docs? (I asked above previously but it got resolved so not sure if you saw it) thanks.

Reply:
@ru-fu chatted with @stgraber this is fine. :)